cs/pagination.texy

```diff
@@ -20,21 +20,23 @@ class ArticleRepository
 {
   use Nette\SmartObject;
 
-  /** @var Nette\Database\Context */
+  /** @var Nette\Database\Connection */
   private $database;
 
 
-  public function __construct(Nette\Database\Context $database)
+  public function __construct(Nette\Database\Connection $database)
   {
     $this->database = $database;
   }
 
 
-  public function findPublishedArticles()
+  public function findPublishedArticles(): \Nette\Database\ResultSet
   {
-    return $this->database->table('articles')
-      ->where('created_at < ', new \DateTime())
-      ->order('created_at DESC');
+    return $this->database->query(
+      'SELECT * FROM `articles`
+       WHERE `created_at < ?
```

Comment:
chybí tu `  nicméně možná bych je uplně vynechal, ať to není těsně svázáno s MySQL.